pep8.py

```diff
@@ -1018,7 +1018,15 @@ def explicit_line_join(logical_line, tokens):
                 parens -= 1
 
 
-def break_around_binary_operator(logical_line, tokens):
+def _is_binary_operator(token_type, text):
+    # The % character is strictly speaking a binary operator, but the
```

Comment:
Minor, but I'd prefer to move this to a `"""..."""` style doc string rather than `#` since we're touching it anyways.